src/F2_autoreport.py

```diff
@@ -650,8 +651,13 @@ def create_app(results_json):
     df_scalar_matrix = convert_scalar_matrix_to_dataframe(results_json)
     df_cost_matrix = convert_cost_matrix_to_dataframe(results_json)
 
-    warnings_dict = parse_simulation_log(log_type="WARNING")
-    errors_dict = parse_simulation_log(log_type="ERROR")
+    output_path = results_json[SIMULATION_SETTINGS][PATH_OUTPUT_FOLDER]
+    warnings_dict = parse_simulation_log(
+        path_log_file=os.path.join(output_path, "mvs_logfile.log"), log_type="WARNING",
```

Comment:
We need to define a constant variable for the log file name

Reply:
Yes :)

Reply:
https://github.com/rl-institut/mvs_eland/issues/516